mikrotik/resource_bgp_instance_test.go

```diff
@@ -49,10 +49,10 @@ func TestAccMikrotikBgpInstance_createFailsOnRouterOSv7(t *testing.T) {
 	routerId := internal.GetNewIpAddr()
 	as := acctest.RandIntRange(1, 65535)
 
-	resource.ParallelTest(t, resource.TestCase{
-		PreCheck:          func() { testAccPreCheck(t) },
-		ProviderFactories: testAccProviderFactories,
-		CheckDestroy:      testAccCheckMikrotikBgpInstanceDestroy,
+	resource.Test(t, resource.TestCase{
```

Comment:
Was changing this to `resource.Test` intentional?

Reply:
No, that's a left over after debugging issue during implementation.